open-source-hosts-plugin/README.md

```diff
@@ -30,6 +30,8 @@ To debug the plugin, press ctrl+alt+I (or cmd+option+I on a mac) to bring up the
 
 ### Add a host to the scene
 
+**Note:** Please disable the SSAO (Scene -> Inspector -> Rendering -> Disable SSAO 2) to fix the lighting issue until this problem gets fixed by BabylonJS Editor.
```

Comment:
Is there an issue we could link to? (Also, any idea when the upgrade to BabylonJS 5.* will drop? We'll definitely want to test our plugins when that happens)

Reply:
Sure. I will add the forum link to the description. The babylonJS 5.0 was officially announced just several minutes before. https://forum.babylonjs.com/t/babylon-js-5-0-has-officially-been-announced/30074 However, the editor integration with the newest BabylonJS is another story. We should definitely test our work compatibility after the new release.